### PR TITLE
Setup React Router and implement links / useHistory

### LINKED ISSUE #31: Page routing
# Description

Route pages, overlays etc.

# Notes and resources

React Router?
React Helmet?
Routing with external log in on Spotify?

### PR DESCRIPTION
**Pull request by ncbecker**
## Description

To setup routes I created the missing pages (these are work in progress). This PR focuses on the initial React Router setup. 

## Related Issue

Closes #31

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ https://xd.adobe.com/view/47c0820e-b7e7-4464-b887-bf6c82e7c496-1293/
 - [concurrently](https://github.com/kimmobrunfeldt/concurrently)
 - [Styled Components](https://styled-components.com/)
 - [React Lottie](https://github.com/chenqingspring/react-lottie)
+- [React Router](https://reactrouter.com/)
```
Comment (phlgr) [inline README.md:28]: Great attribution list

File: client/src/pages/EpisodeDetailsPage.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const EpisodeDetailsPage = () => {
+  return (
+    <>
+      <div>EpisodeDetailsPage</div>
+    </>
```
Comment (phlgr) [inline client/src/pages/EpisodeDetailsPage.js]:
```suggestion
  return (

      <div>EpisodeDetailsPage</div>
    
```

**Review by phlgr — COMMENTED** [review]
Very clean and readable code 🤓

**Comment by ReneSchmitz** [comment]
Good job! 🤙🏻

File: README.md (hunk 1 above)
Comment (Sina-Sche) [reply README.md:28]: Agree with Philipp! Great idea 🥇

**Review by Sina-Sche — APPROVED** [review]
lgtm! :)

**Pull request merged**